### PR TITLE
[MINI-5697] Bug Fix for unable to open any contact from settings 

### PR DESCRIPTION
**Pull request by GondiTilak**
# Description
Issues found in dev testing, On editing the contacts the empty values for the fields were update to UI on close action.

## Links
jira: https://jira.rakuten-it.com/jira/browse/MINI-5697
Fix Demo: https://rak.box.com/s/i8pxv9ek871kviqcjmuiv7firaiv50mi

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by laubengaier — APPROVED** [review]
lgtm! 👍

File: Sources/Classes/core/Protocols/MiniAppUserInfoDelegate.swift [hunk 1]
```diff
@@ -72,7 +72,7 @@ public extension MiniAppUserInfoDelegate {
 }
 
 /// Contact Object of a User
-public class MAContact: Codable, Equatable, Hashable, Identifiable {
+public struct MAContact: Codable, Equatable, Hashable, Identifiable {
```
Comment (rleojoseph) [inline Sources/Classes/core/Protocols/MiniAppUserInfoDelegate.swift:75]: Any reason why this was changed to `struct ` from `class` ?

Comment (GondiTilak) [reply Sources/Classes/core/Protocols/MiniAppUserInfoDelegate.swift:75]: This was changed as the class was having issues with the editing function, the bonded object was updated  and changed in the UI even before we tap on save or cancel buttons. So made it as struct to use it as pass by value.

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Review by rleojoseph — CHANGES_REQUESTED** [review]
Looks good to me :+1: Just fix the codecov and fastlane issue

**Review by rleojoseph — APPROVED** [review]

**Pull request merged**